specification/netapp/resource-manager/Microsoft.NetApp/stable/2020-12-01/examples/Accounts_CreateOrUpdate.json

```diff
@@ -0,0 +1,50 @@
+{
+  "parameters": {
+    "subscriptionId": "D633CC2E-722B-4AE1-B636-BBD9E4C60ED9",
+    "resourceGroupName": "myRG",
+    "accountName": "account1",
+    "api-version": "2020-11-01",
```

Comment:
>"api-version": "2020-11-01", [](start = 4, length = 28)

Please fix API versions across all the files.

Reply:
sorry about that, done